metadata-ingestion/src/datahub/ingestion/source/powerbi/rest_api_wrapper/powerbi_api.py

```diff
@@ -183,6 +186,11 @@ def fill_tags() -> None:
         return reports
 
     def get_workspaces(self) -> List[Workspace]:
+        if self.__config.modified_since:
+            workspaces = self.get_modified_workspaces()
+            if workspaces:
```

Comment:
why this if ?if get_modified_workspaces returns empty list then we don't need this